lib/ProducerConfiguration.cc

```diff
@@ -16,6 +16,8 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+#include "pulsar/ProducerConfiguration.h"
```

Comment:
```suggestion
#include <pulsar/ProducerConfiguration.h>
```

We usually use `<>` for headers of the Pulsar client itself.